Add support for output directory in extraction (#16126)
coq/coq: doc/changelog/08-vernac-commands-and-options/16126-extraction-output.rst

- **Added:**
  :opt:`Extraction Output Directory` option for specifying the

>   directory in which `.ml` and `.mli` files will be appear during
>   extraction

Reviewer: ```suggestion
  directory where `.ml` and `.mli` files will be written for extractions
```

Author: Since we talk about the process "of extraction" I didn't want to change the wording at all. Keeping it nouny rather than verby.

Reviewer: "files will be appear during extraction" in ungrammatical.  Please fix.
